s3/src/main/scala/akka/stream/alpakka/s3/impl/S3Stream.scala

```diff
@@ -78,9 +84,12 @@ private[alpakka] final class S3Stream(credentials: AWSCredentials, region: Strin
     import mat.executionContext
 
     val req = HttpRequests.initiateMultipartUploadRequest(s3Location, contentType, cannedAcl)
+
     val response = for {
       signedReq <- Signer.signedRequest(req, signingKey)
+      _ <- Future(system.log.info("Request: {}", signedReq))
       response <- Http().singleRequest(signedReq)
+      _ <- Future(system.log.info("Response: {}", response))
```

Comment:
why spawn a future for doing the logging?
what is the `toString` representation of the request and response? can it be big, and therefore inappropriate for logging?